docs/graphics-math/fireworks.md

```diff
@@ -0,0 +1,418 @@
+# Fireworks!
+
+Enjoy the fireworks! Watch them being launched in the background, and even
+create some of your own by pressing buttons.
+
+```typescript
+/**
+ * particle effects
+ * 
+ * The particle effects you see in arcade are formed through the combination
+ * of several objects:
+ * 
+ * **Particles** are the individual elements you see on the screen -
+ * they contain a small amount of information that conveys where and how they are represented.
+ * 
+ * **ParticleFactories** describe how the particles are created
+ * and how they should be be displayed on the screen. This sets things like initial speed,
+ * color, and how long the particle survives
+ * 
+ * **ParticleSources** control when Particles are allowed to be created,
+ * they apply updates to the particles, they display them to the screen,
+ * and make sure the state is clean (e.g. getting rid of particles when their lifespan runs out).
+ * They use ParticleFactories to handle the creation of the sprites involved, so a single source
+ * can have a wide variety of actual behaviors by attaching different types of factories to it.
+ * Most effects in Arcade use the standard particle source, but some use slightly modified versions
+ * to add special behaviors. In particles, for example, the Fire effect has a special source that makes
```

Comment:
…In the particles namespace, for example....

Reply:
I think I meant to write `in particular`, and just got mixed up momentarily hah! Either way works